connect-id-example/proguard-rules.pro

```diff
@@ -40,4 +40,21 @@
 -keep class okio.** { *; }
 -dontwarn java.nio.file.*
 -dontwarn org.codehaus.mojo.animal_sniffer.IgnoreJRERequirement
--dontwarn okio.**
\ No newline at end of file
+-dontwarn okio.**
+
+-keep class * extends java.util.ListResourceBundle {
+    protected Object[][] getContents();
+}
+
+-keep public class com.google.android.gms.common.internal.safeparcel.SafeParcelable {
+    public static final *** NULL;
+ }
+
+-keepnames @com.google.android.gms.common.annotation.KeepName class *
+-keepclassmembernames class * {
+    @com.google.android.gms.common.annotation.KeepName *;
+}
+
+-keepnames class * implements android.os.Parcelable {
+    public static final ** CREATOR;
+}
```

Comment:
N00b question: How do you get this list of things for the `proguard-rules.pro` file?

Reply:
Not an easy task :)
Some googling, trial and error. Usually library devs publish their proguard-rules or repos like
https://github.com/krschultz/android-proguard-snippets/tree/master/libraries